File: audiform.client/src/types/order.ts
export type OrderStatus =
  | 'nieuw'
  | 'ontvangen'
  | 'in productie'
  | 'gereed'
  | 'verzonden'
  | 'afgeleverd'  
  | 'geannuleerd'
  | 'traditioneel'
  | '3D print'
  | 'afgewezen'


export interface OrderState {
  id: number;
  stateName: string;
}

export interface Order {
  id: number;
  orderNumber: string
  patientName: string | null;
  patientNumber: string
  orderDate: string;
  deliveryDate: string;
  state: OrderState;
}

File: Domain/Entities/Order.cs
namespace Domain.Entities;

public sealed class Order
{
    public int Id { get; init; }
    public string OrderNumber { get; init; }
    public string? PatientName { get; init; }
    public string PatientNumber { get; init; }
    public DateTime? OrderDate { get; init; }
    public DateTime? DeliveryDate { get; init; }
    public string? Status { get; init; }
}
